torch/csrc/autograd/input_buffer.cpp

```diff
@@ -16,7 +16,6 @@ void InputBuffer::add(size_t pos, Variable var) {
   if (!old_var.defined()) {
     buffer[pos] = std::move(var);
   } else {
-    AutoGPU auto_gpu(var);
```

Comment:
This is needed

Reply:
I think the AutoGPUStream in EvaluateFunction makes this redundant?

Reply:
Good point. Never mind my comment.